src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.CodeModification/CodeChange/CodeChangeHelper.cs

```diff
@@ -2,17 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 using System.Text.Json.Serialization;
 
-namespace Microsoft.DotNet.Scaffolding.Helpers.Roslyn;
+namespace Microsoft.DotNet.Scaffolding.CodeModification;
 
 [JsonConverter(typeof(JsonStringEnumConverter))]
-internal enum CodeChangeType
+public enum CodeChangeType
```

Comment:
I know you didn't combine them in this PR, but could we split these into one class/enum per file?

Reply:
yep will do, adding a TODO for it.